app.json

```diff
@@ -1,106 +1,129 @@
+    },
+    "AWS_SECRET_ACCESS_KEY": {
+      "description": "AWS Secret Key for S3 storage."
+    },
+    "AWS_STORAGE_BUCKET_NAME": {
+      "description": "S3 Bucket name."
+    },
+    "EDXORG_BASE_URL": {
+      "description": "The base URL of the edX instance to use for logging in.",
+      "required": true
+    },
+    "EDXORG_CLIENT_ID": {
+      "description": "The OAuth client ID configured in the edX instance.",
+      "required": true
+    },
+    "EDXORG_CLIENT_SECRET": {
+      "description": "The OAuth client secret configured in the edX instance.",
+      "required": true
+    },
+    "ELASTICSEARCH_HTTP_AUTH": {
+      "descriptin": "Basic auth settings for connecting to Elasticsearch"
+    },
+    "ELASTICSEARCH_INDEX": {
+      "description": "Index to use on Elasticsearch"
+    },
+    "ELASTICSEARCH_URL": {
+      "description": "URL for connecting to Elasticsearch cluster"
+    },
+    "MAILGUN_KEY": {
+      "description": "The token for authenticating against the Mailgun API"
+    },
+    "MICROMASTERS_ADMIN_EMAIL": {
+      "description": "E-mail to send 500 reports to.",
+      "required": false
+    },
+    "MICROMASTERS_EMAIL_HOST": {
+      "description": "Outgoing e-mail settings",
+      "required": false
+    },
+    "MICROMASTERS_EMAIL_PASSWORD": {
+      "description": "Outgoing e-mail settings",
+      "required": false
+    },
+    "MICROMASTERS_EMAIL_PORT": {
+      "description": "Outgoing e-mail settings",
+      "required": false,
+      "value": "587"
+    },
+    "MICROMASTERS_EMAIL_TLS": {
+      "description": "Outgoing e-mail settings",
+      "required": false,
+      "value": "True"
+    },
+    "MICROMASTERS_EMAIL_USER": {
+      "description": "Outgoing e-mail settings",
+      "required": false
+    },
+    "MICROMASTERS_ENVIRONMENT": {
+      "description": "The execution environment that the app is in (e.g. dev, staging, prod)"
+    },
+    "MICROMASTERS_FROM_EMAIL": {
+      "value": "MITx MicroMasters <micromasters-support@mit.edu>"
+    },
+    "MICROMASTERS_SECURE_SSL_REDIRECT": {
+      "description": "Application-level SSL redirect setting.",
+      "required": false,
+      "value": "True"
+    },
+    "MICROMASTERS_USE_S3": {
+      "description": "Use S3 for storage backend (required on Heroku)",
+      "value": "True"
+    },
+    "NEW_RELIC_APP_NAME": {
+      "default": "MicroMasters",
+      "description": "Application identifier in New Relic."
+    },
+    "OPEN_EXCHANGE_RATES_APP_ID": {
+      "description": "The app ID for the open exchange rates API"
+    },
+    "OPEN_EXCHANGE_RATES_URL": {
+      "value": "https://openexchangerates.org/api/"
+    },
+    "SECRET_KEY": {
+      "description": "Django secret key.",
+      "generator": "secret"
+    },
+    "SENTRY_DSN": {
+      "description": "The connection settings for Sentry"
+    },
+    "STATUS_TOKEN": {
+      "description": "Token to access the status API.",
+      "required": true
     }
+  },
+  "keywords": [
```

Comment:
Those were already there, I just sorted the whole JSON object so it messed with the diff. I'm honestly not sure what they're for.